main/src/library/Benchmark.flix

```diff
@@ -20,7 +20,7 @@ namespace Benchmark {
     /// impure since timing is considered a side-effect in this context.
```

Comment:
Should this like be `IO since timing is considered a side-effect in this context.` now ?

Reply:
Yes